index.html

```diff
@@ -1739,8 +1764,16 @@ <h3>The <dfn>subscribeEvent()</dfn> method</h3>
         Takes as arguments |eventName:string|, |listener:WoTListener| and
         optionally  |onerror:ErrorListener| and |options:InteractionOptions|.
         It returns a {{Promise}} to signal success or failure.
+        <p class="ednote">
+          This algorithm allow for only one active {{Subscription}} per <a>Event</a>. If a new
```

Comment:
Should we repeat this every time, or just say once for the whole API?

Reply:
Good point. I think another good place to state this is in the `Subscription` section but I don't know if we want to keep this simple to possibly change it in the future.

Reply:
We can also omit the ednote, since the algorithm is clear enough.

Reply:
typo?: this algorithm allowS ...